source/bin/tvm.rs

```diff
@@ -45,14 +45,15 @@ enum ExitCode {
     InvalidOption,
     MissingFile,
     InvalidFile,
+    MultipleFiles,
     Panic
 }
 
 fn usage() -> String {
-    "Usage: tvm [options] [file]\n".to_string() +
-    "Options:\n" +
-    "    -v, --version    Print version.\n" +
-    "    -h, --help       This help."
+    "Usage: tvm [options] [file]\
```

Comment:
But:
```
foo\n\
bar
```
is strictly equivalent to:
```
foo\
\nbar
```
isn't it?

If yes, I prefer the former (because the EOL belongs to the line, not to the next line).